pagopa/io-app
# fix: remove header from `IdpSuccessfulAuthentication`

## Short description
This PR removes the screen header from `IdpSuccessfulAuthentication`

<details open><summary>Details</summary>
<p>

| ❌ | ✅ |
|--------|--------|
| <img src="https://github.com/user-attachments/assets/42c6c7ec-493b-4970-9d98-3abdae756e5a" width="250" /> | <img src="https://github.com/user-attachments/assets/0f6fc215-40ca-43eb-af51-2dd444c95c46" width="250" /> | 

</p>

</details> 

## How to test
Try to login and check that `IdpSuccessfulAuthentication` presents itself with no header.
